### PR TITLE
Refactored AppSaving store

### PR DESCRIPTION
**Pull request by minotogna**
- AppSaving store refactor
- **[Added Redux store doc](https://github.com/openforis/arena/pull/858/files#diff-dedbad1c381d2b0b32d702b036888056)**

### CODE REVIEW

File: doc/reduxStore.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## **Redux store**
+
+The order shows the hierarchy
+
+- **system**
+  - i18n
+  - systemStatus
+  - system error
```
Comment (sorja) [inline doc/reduxStore.md:8]: typo

Comment (minotogna) [reply doc/reduxStore.md]: fixed

File: webapp/store/app/saving/actions.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Counter from '@core/counter'
+
+export const APP_SAVING_UPDATE = 'app/saving/update'
+const counter = new Counter()
```
Comment (ramirobg94) [inline webapp/store/app/saving/actions.js:4]: why do we need this counter?

Comment (minotogna) [reply webapp/store/app/saving/actions.js:4]: checking how many times the use clicks on saving

File: webapp/store/app/saving/state.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import * as R from 'ramda'
+
+import * as AppState from '../state'
+
+export const stateKey = 'saving'
+const getState = R.pipe(AppState.getState, R.prop(stateKey))
+
+export const assocSaving = (saving) => saving
+
+export const isSaving = R.pipe(getState, R.equals(true))
```
Comment (sorja) [inline webapp/store/app/saving/state.js:10]: Maybe we can re-evaluate the need for Ramda in some cases?

**Pull request merged**